package-lock.json

```diff
@@ -0,0 +1,5212 @@
+{
```

Comment:
:pray: Looks like yarn.lock has not been updated. Please use yarn to manage dependencies for this project. I'm happy to include package-lock.json for npm users, but yarn should be used for development.

Reply:
Oops, sorry as you can tell I just used npm for installing packages.

This does raise the question though, do you want to keep using `yarn`? `npm` seems pretty good these days. Something to think about.

Either way, I'll fix this up and if you want to move back it can be tackled in another PR.

Reply:
I've experienced many issues from regressions in `npm` over the years, and no such issues from `yarn`. All I can surmise is that the `npm` developers do not have a comprehensive test suite. I have more confidence in `yarn` working as expected between versions. It's also unfortunate that the version of `npm` is tied to the version of `node`, it makes using tools like `nvm` difficult.